recipe/meta.yaml

```diff
@@ -44,122 +44,115 @@ build:
 # Shotgun strat on requirements until we can narrow them down
 requirements:
   build:
-  - armadillo==8.200.0
-  - blas==1.1
-  - bullet==2.86.1
-  - bz2file==0.98
-  - bzip2==1.0.6
-  - cmake=>3.10
-  - cspice==66
-  - curl==7.60.0
-  - doxygen==1.8.14
-  - eigen==3.3.3
-  - embree==2.14.0
-  - geos==3.5.1
-  - geotiff==1.4.2
-  - gmm==5.0
-  - gmp==6.1.2
-  - gsl==2.2.1
-  - hdf5==1.8.18
-  - icu==58.2
-  - jama==125
+  - ale >=0.7.1
+  - boost=1.68.0
+  - armadillo
+  - blas
+  - bullet=2.86.1=0
+  - bz2file
+  - bzip2
+  - cmake >=3.15
+  - cspice
+  - curl
+  - eigen
+  - embree
+  - geos>=3.7,<3.8
+  - geotiff
+  - gmm
+  - gmp
+  - gsl
+  - hdf5
+  - icu
+  - jama
   - jpeg==9b
   - kakadu==1
-  - krb5==1.14.2
+  - krb5
   - libpng>=1.6.34
-  - libprotobuf==3.5.2
-  - libtiff=>4.0.9
-  - libxml2==2.9.7
+  - libprotobuf=3.9.1
+  - libtiff
+  - libxml2
   - make
-  - mesalib==17.2.0
-  - mysql==5.7.20
-  - mysql-connector-c==6.1.6
-  - nanoflann==1.2.2
+  - mesalib
+  - mysql
+  - mysql-connector-c
+  - nanoflann
+  - nlohmann_json
   - ninja==1.7.2
   - nn
-  - openblas==0.2.19
-  - opencv
-  - openssl==1.0.2n
-  - pcl==1.8.1
-  - pip==9.0.1
-  - protobuf==3.5.2
-  - python==3.6
-  - qhull==7.2.0=0
+  - opencv>=3.0.0,<=4.0.0
+  - openssl>=1.0.0,<1.1.0
+  - pcl
+  - protobuf=3.9.1
+  - qhull
   - qt=5.9.6
   - qwt=6.1.3
-  - setuptools=38.5.1
-  - sip==4.18
-  - sqlite==3.13.0
-  - suitesparse==4.5.4
-  - superlu==5.2.1
-  - tnt==126=0
-  - wheel==0.30.0
-  - x264==20131218
-  - xalan-c==1.11
-  - xerces-c==3.1.4
-  - xorg-kbproto==1.0.7
+  - sqlite
+  - suitesparse
+  - superlu
+  - tnt
+  - x264=20131218
+  - xerces-c
+  - xorg-kbproto
   - xorg-libice
   - xorg-libsm
-  - xorg-libx11==1.6.4
+  - xorg-libx11
   - xorg-libxi
-  - zlib==1.2.11
+  - zlib
+
   run:
-  - armadillo==8.200.0
-  - blas==1.1
-  - bullet==2.86.1
-  - bz2file==0.98
-  - bzip2==1.0.6
-  - cspice==66
-  - curl==7.60.0
-  - doxygen==1.8.14
-  - eigen==3.3.3
-  - embree==2.14.0
-  - geos==3.5.1
-  - geotiff==1.4.2
-  - gmm==5.0
-  - gmp==6.1.2
-  - gsl==2.2.1
-  - hdf5==1.8.18
-  - icu==58.2
-  - jama==125
+  - ale >=0.7.1
+  - boost=1.68.0
+  - armadillo
+  - blas
+  - bullet=2.86.1=0
+  - bz2file
+  - bzip2
+  - cspice
+  - curl
+  - doxygen
+  - eigen
+  - embree
+  - geos>=3.7,<3.8
+  - geotiff
+  - gmm
+  - gmp
+  - gsl
+  - hdf5
+  - icu
+  - jama
   - jpeg==9b
   - kakadu==1
-  - krb5==1.14.2
+  - krb5
   - libpng>=1.6.34
-  - libprotobuf==3.5.2
-  - libtiff=>4.0.9
-  - libxml2==2.9.7
-  - mesalib==17.2.0
-  - mysql==5.7.20
-  - mysql-connector-c==6.1.6
-  - nanoflann==1.2.2
+  - libprotobuf=3.9.1
+  - libtiff
+  - libxml2
+  - mesalib
+  - mysql
+  - mysql-connector-c
+  - nanoflann
+  - nlohmann_json
   - conda-forge/label/gcc7::nn
-  - openblas==0.2.19
-  - opencv
-  - openssl==1.0.2n
-  - pcl==1.8.1
-  - pip==9.0.1
-  - protobuf==3.5.2
-  - python==3.6
```

Comment:
We may need to keep this in. I'm not 100% sure, needs testing.

Reply:
I'll run local tests on this without it in and report back (this will take some time.)

Reply:
Update: Didn't even get to the stage of running local tests. Building this, pushing it up to my personal channel, installing it locally, and running any app results in "cant find opencv library" type errors.

Reply:
@jessemapel We have `conda create -n isis3 python=3.6` as one of the first steps in our "install ISIS" instructions. Does this effectively accomplish the same thing?

Reply:
yes